src/methods/method_overloading/medium/q6/OverloadFloat.java

```diff
@@ -0,0 +1,13 @@
+package methods.method_overloading.medium.q6;
+
+/*
+ Overload a Java Method so that it can take a double variable and float variable or it can take a int variable and long
```

Comment:
We should make the question text better here and give the student the name of the method

Reply:
ALso we don't specify the type it should return